i18n/locales/it.json

```diff
@@ -489,5 +489,8 @@
   },
   "modInputField": {
     "optional": "Optional"
+  },
+  "searchResultsItem": {
+    "lastUpdate": "Last update: "
```

Comment:
nit: I would personally put the `:` and space in the string you're compiling in javascript instead of the translation for combined readability, but it's a weak argument 😅